src/backend/modules/company/company.service.ts

```diff
@@ -47,9 +47,13 @@ export class CompanyService {
           0,
         ),
         priceCents:
-          result.UserStock.flatMap((us) =>
-            us.SellOffer.map((so) => so.unitSellPriceCents),
-          ).reduce((all, cur) => (all < cur ? all : cur)) ?? null,
+          result.UserStock.length > 0
+            ? Math.min(
+                ...result.UserStock.flatMap((us) =>
+                  us.SellOffer.map((so) => so.unitSellPriceCents),
+                ),
+              )
+            : null,
```

Comment:
I made this changes in mine branch (97) as well. I changed null to 0 though, because I needed it to be defined.

Reply:
Why?